hal/network/lwip/wiznet/wiznetif.cpp

```diff
@@ -72,15 +73,15 @@ LOG_SOURCE_CATEGORY("net.en")
             break;                                                              \
         }                                                                       \
         /* I'd really love to get rid of this */                                \
-        os_thread_yield();                                                      \
+        HAL_Delay_Milliseconds(1);                                              \
```

Comment:
Why switch over to a delay here?

Reply:
`os_thread_yield()` yields only to threads with higher priority. `HAL_Delay_Milliseconds(1)` will also yield to threads of lower or same priority if they require CPU time.

Reply:
Priority inversion problem?

Reply:
No, that's the documented behavior: https://www.freertos.org/a00020.html#taskYIELD

Reply:
ok, sounds good